src/MultiplexingDistributedCache/MultiplexingDistributedCache.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Caching.Distributed;
+
+namespace Aqovia.Cache
+{
+    public class MultiplexingDistributedCache : IDistributedCache
+    {
+        private readonly IDistributedCache _primary;
+        private readonly IDistributedCache _secondary;
+        public MultiplexingDistributedCache(IDistributedCache primary, IDistributedCache secondary = default)
+        {
+            _primary = primary;
+            _secondary = secondary;
+        }
+
+        public byte[] Get(string key)
+        {
+            return _primary.Get(key);
+        }
+
+        public async Task<byte[]> GetAsync(string key, CancellationToken token = default)
+        {
+            return await _primary.GetAsync(key, token).ConfigureAwait(false);
+        }
+
+        public void Refresh(string key)
```

Comment:
I couldn't find any documentation on what "Refresh" actually does and therefore couldn't be sure whether it should operate on primary-only  or both caches.

Reply:
From [here](https://learn.microsoft.com/en-us/aspnet/core/performance/caching/distributed?view=aspnetcore-7.0#idistributedcache-interface):
> [Refresh](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.caching.distributed.idistributedcache.refresh), [RefreshAsync](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.caching.distributed.idistributedcache.refreshasync): Refreshes an item in the cache based on its key, resetting its sliding expiration timeout (if any).

I think it should do it on both.